### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 40

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#216] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255279-d98958ba-3c95-4fe8-8c30-de02ca4c35ca.png)

**Comment by i0nut02** [comment]
1. (p + q -1 scelgo q-1)
2. (p - q + q -1 scelgo q-1) --> (p-1 scelgo q-1)
3. 29022003  vedo se ci sono ripetizioni e le elimino 2903    quindi avrò   4* 3* 2

**Comment by mildora00** [comment]
Buonasera, qualcuno sa spiegarmi perché @i0nut02  nel secondo esercizio ha inserito anche il - q?

**Comment by i0nut02** [comment]
@mildora00 il -q nell'esercizio 2 è perché il prof non considera 0 un numero intero positivo, quindi dobbiamo portare tutte le x come minimo a 1 ad esempio con q = 5
1 + 1 + 1 + 1 + 1 = p-q
